component-overview/examples/typography/Heading.jsx

```diff
@@ -9,9 +9,12 @@ import {
 
 <div>
     <Heading1>Overskrift nivå 1</Heading1>
-    <Heading2>Overskrift nivå 2</Heading2>
-    <Heading3>Overskrift nivå 3</Heading3>
-    <Heading4>Overskrift nivå 4</Heading4>
-    <Heading5>Overskrift nivå 5</Heading5>
-    <Heading6>Overskrift nivå 6</Heading6>
+        <Heading2>Overskrift nivå 2</Heading2>
```

Comment:
Burde man kallat det noe annet en "Overskrift nivå 2"? For nå blir det 2 headings som er helt lik. Kanskje ikke har noe og si før dokumentasjonen?

Reply:
request change er basert på commiten ikke på dette. Dette er kun til vurdering :)